spec/docs/logicalSource.md

```diff
@@ -0,0 +1,11 @@
+# Defining Logical Sources
+
+A <dfn>logical source</dfn> is a function on a [=data source=] that provides data to be mapped to [=RDF triples=].
```

Comment:
Is it actually a function?

Reply:
Yeah maybe we need to define it more abstractly. How would you define it?

Reply:
> `A Logical Source is an abstract description for describing data access and iteration of a data source which is mapped to RDF quads`.

I then suggest: 
```suggestion
A <dfn>logical source</dfn> is an abstract construct to describe data access and iteration for a [=data source=] such that it can be mapped to [=RDF triples=].
```

> * Logical Source is a term -> caps

I prefer to keep it lower case, but have all occurrences of terms linked to the definitions. Otherwise there will be a lot of caps in sentences, which IMO does not aid readability

> * RDF quads -> triples implies no Named Graph support

I know @andimou wants to keep the text close to the [R2RML rec](https://www.w3.org/TR/r2rml). There the term RDF triple is used throughout. Also, I'm not sure there is an authorative definition of quad anywhere in the W3C recs.

Reply:
I like the suggested change! The word 'function' is tricky as it might confuse people with rml-fnml.

> I prefer to keep it lower case, but have all occurrences of terms linked to the definitions.

That's something we need to align across all specs. RML-IO prefers caps when it talks about classes like Logical Source, following the practices used in ontologies.

> There the term RDF triple is used throughout. 

Yeah that's fine for me as well, aligns with Triples Maps though.

Reply:
> That's something we need to align across all specs. RML-IO prefers caps when it talks about classes like Logical Source, following the practices used in ontologies.

OK. Core is quite heavy on definitions and references to defined terms. So I think it's more of an issue there.

Reply:
It would be great if we can coordinate across all specs the same approach to not confuse readers?